engineering/methodology.md

```diff
@@ -1,131 +1,196 @@
-# Engineering Methodology
-
-Date: 2017-05-09
+# Engineering methodology
+
+Date: 2018-01-15
+
+* [Intro](#intro)
+* [OKRs](#okrs)
+* [The Big Change](#the-big-change)
+* [Kanban](#kanban)
+  * [Company Kanban](#company-kanban)
+  * [Team Kanban](#team-kanban)
+    * [How to pick next issues to worked on?](#how-to-pick-next-issues-to-worked-on)
+* [Leads Meeting](#leads-meeting)
+* [Team Meeting](#team-meeting)
+* [Standup](#standup)
+* [Demo](#demo)
+* [Keeping track of things](#keeping-track-of-things)
+* [Definitions](#definitions)
+  * [What does TODO mean?](#what-does-todo-mean)
+  * [What does Done mean?](#what-does-done-mean)
+  * [What does Blocker mean?](#what-does-blocker-mean)
 
 ## Intro
 
-These are the notes from
- - 2016-09-12 Scrum/OKR
- - 2017-05-08 Sprint #13 Retrospective
+These are the notes from:
+
+* 2017-01-09 Scrum of Scrums
+* 2017-01-10/11 Workflow discussions
+
+Meetings for the Engineering team leads at source{d}.
+It’s the replacement for the [previous workflow](https://github.com/src-d/guide/blob/abe7932e3bc58898e49a5c1c0e3e2cac6d387144/engineering/methodology.md) \w SCRUM
+
+## OKRs
+
+Company publishes OKRs every quarter.
+
+2018Q1 OKRs in https://github.com/src-d/guide/blob/general/OKRs/README.md
```

Comment:
I think this link will be broken once the OKR are merged
It swould include the branchname, commit hash or HEAD after `\blob\`

Reply:
Its exactly the opposite, the link will be working once the #97 get merged.

Reply:
really?
I mean...
[https://github.com/src-d/guide/blob/**HEAD**/README.md](https://github.com/src-d/guide/blob/master/README.md) &rarr; works
https://github.com/src-d/guide/blob/README.md &rarr; broken
Because `/master` (or refference name) is missing after `/blob`

so... I guess why the following link does not need the reference name after `/blob`?
https://github.com/src-d/guide/blob/general/OKRs/README.md